### PR TITLE
Remove update time text after manual update

### LINKED ISSUE #28: Forcing update while scheduled doesn't remove scheduled update
I was testing this out using the Query Monitor plugin. Repro:
- Install version [3.5.1](https://downloads.wordpress.org/plugin/query-monitor.3.5.1.zip) from a zip file
- Enable auto-updates
- The update option is still available to force the update. Click it, and notice that the scheduled update appears to still linger

still scheduled after forced update

### PR DESCRIPTION
**Pull request by bookdude13**
Fixes #28. 

Note that in #28 the issue disappeared after a refresh, since the page reloading set the text properly. I didn't think a page reload made sense after an update (especially since multiple updates may occur with bulk actions), so I added a jQuery hook. If there are coding standard changes, or a better place to put this, let me know and I can update it. I'm always ready to learn 😄

### CODE REVIEW

File: wp-autoupdates.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
 Tested up to: 5.4
 Author: The WordPress Team
 Author URI: https://wordpress.org
-Contributors: wordpressdotorg, audrasjb, whodunitagency, pbiron, xkon, karmatosed, mapk
+Contributors: wordpressdotorg, audrasjb, whodunitagency, pbiron, xkon, karmatosed, mapk, bookdude13
```
Comment (audrasjb) [inline wp-autoupdates.php]: Please don’t directly add your name to the contributors on the plugin header.
You can add your name on the readme.md file and I will take care your name is added to the props list of WordPress 5.5, but the plugin’s list of contributors is updated manually.

Thanks for your comprehension :)

**Review by audrasjb — CHANGES_REQUESTED** [review]

File: wp-autoupdates.php (hunk 1 above)
Comment (bookdude13) [reply wp-autoupdates.php]: My bad, and understood :) I thought I saw something similar in a PR here, but must've been mistaken.

Comment (audrasjb) [reply wp-autoupdates.php]: Thank you very much :)

**Review by audrasjb — DISMISSED** [review]

**Review by pbiron — CHANGES_REQUESTED** [review]
The `</span>` should be output before the `<br />`.

**Review by pbiron — CHANGES_REQUESTED** [review]
Setting the text to the empty string works.   but I think it would be better to just remove the span (e.g., `plugin_auto_update_time_text.remove()`.

**Comment by bookdude13** [comment]
> it would be better to just remove the span

I agree, removed.

> The `</span>` should be output before the `<br />`.

If this happens then there is a gap between the auto-updates enabled text and the enable/disable link (see Hello Dolly):
gap_with_br

I could try to use other selectors to remove that `<br />` in the script separately, but in my eyes the `<br />` is part of the text I'm trying to select with the `<span>`. If that's just bad practice though let me know.

**Comment by pbiron** [comment]
> If this happens then there is a gap between the auto-updates enabled text and the enable/disable link (see Hello Dolly):

You're right, my bad...I reviewed too quickly. 

@audrasjb I think this is good to go now.

**Review by pbiron — APPROVED** [review]

**Comment by audrasjb** [comment]
Yeah, it's good to go. Thank you @bookdude13 👍

**Pull request merged**